resources/js/Pages/Praktikan.vue

```diff
@@ -1667,9 +1672,89 @@ export default {
             }
             
             // Start opening Jurnal Section and get all SOAL from
-            // get soal from soal__jurnals and soal__fitbs
+            // get soal from soal__jurnals and soal__fitbs 
+            // new: get jawaban from jawaban__tp where soal__tp.isProgram=1
+            globe.tpPickRandomProgram();
+            globe.$axios.post('api/getTp/'+globe.currentUser.nim+'/'+globe.current_praktikum.modul_id).then(response => {
+            globe.soalPresentasi = JSON.stringify(response.data.all_tp[globe.randomNumber].jawaban)
```

Comment:
Change this after changing the `tpPickRandomProgram()` function above
```suggestion
            randomNumber = globe.tpPickRandomProgram();
            globe.$axios.post('api/getTp/'+globe.currentUser.nim+'/'+globe.current_praktikum.modul_id).then(response => {
            globe.soalPresentasi = JSON.stringify(response.data.all_tp[randomNumber].jawaban)
```